docs/eBpfExtensions.md

```diff
@@ -296,14 +296,73 @@ The helper function ID for a general helper function must be in the range 0 - 65
 The parameter and return types for these helper functions must adhere to the `ebpf_argument_type_t` and
 `ebpf_return_type_t` enums.
 
-### 2.7 Registering Program Types and Attach Types
+### 2.7 Registering Program Types and Attach Types - eBPF Store
 The eBPF Execution Context loads an eBPF program from an ELF file that has program section(s) with section names. The
 prefix to these names determines the program type. For example, the section name `"xdp"` implies that the corresponding
-program type is `EBPF_PROGRAM_TYPE_XDP`. The Execution Context discovers the program type associated with a section
-prefix by reading the data from Windows registry. When an eBPF extension is installed, it must update the registry with
-the program types it implements along with the associated section prefixes.
-
-_Note: The registry location and data format are TBD. This is currently tracked by issue #223._
+program type is `EBPF_PROGRAM_TYPE_XDP`.
+
+The *Execution Context* discovers the program type associated with a section prefix by reading the data from Windows registry,
+in what is referred to aa the ***"eBPF Store"***.
+When an eBPF extension is installed, it must update the eBPF store with the program types it implements along with the associated section prefixes.
+
+To operate on the eBPF store, the extension must link the `\build\native\lib\ebpf_store_helper_km.lib` kernel-mode library and include the related `\build\native\include\ebpf_store_helper.h` header file, both distributed within the [eBPF for Windows NuGet package](https://www.nuget.org/packages/eBPF-for-Windows/). With these, the extension can use the following APIs to register program types, attach types and helper functions:
```

Comment:
This mentions "build\native" but is that really the correct path when you install the development module in the MSI?

Reply:
that is actually the path un the nuget... but I agree that just mentioning the relative paths from the root of the actual package would be better, i.e. \lib and \include